config/helpers.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const path = require('path');
+const fs = require('fs');
+
+const localConfigFileName = 'config.dev.json';
+const startRemoteClientSecret = 'bWlncmF0aW9ucy5vcGVuc2hpZnQuaW8K';
+
+const getLocalConfig = () => {
+  const configPath = path.join(__dirname, localConfigFileName);
+  if (!fs.existsSync(configPath)) {
+    console.error('ERROR: config/config.dev.json is missing');
+    console.error(
+      'Copy config/config.dev.json.example to config/config.dev.json' +
+        ' and optionally configure your dev settings. A valid clusterUrl is ' +
+        ' required for start:remote.'
+    );
+    process.exit(1);
+  }
+  const localConfig = require(configPath);
+  const migMeta = {
+    clusterApi: localConfig.clusterApi,
+    oauth: {
+      clientId: localConfig.oauthClientId,
+      redirectUri: localConfig.redirectUri,
+      userScope: localConfig.userScope,
+      clientSecret: startRemoteClientSecret,
```

Comment:
Looks like we can just use `localConfig.oauthClientSecret` here instead of having the `startRemoteClientSecret` variable duplicating that value

Reply:
done